features/generate-7.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Generate a new project using Tuist (suite 7)
+
+Scenario: The project is an macOS command line tool without any dependencies (command_line_tool_basic)
+    Given that tuist is available
+    And I have a working directory
+    Then I copy the fixture command_line_tool_basic into the working directory
+    Then tuist generates the project
+    Then I should be able to build for macOS the scheme CommandLineTool
```

Comment:
@olejnjak one final suggestion I'd make here is: what about adding the following extra step:
```
Then I should be able to run the tool X
```
With that, we'd test that we've got a valid binary with all its dependencies linked.

Reply:
Okay, but that will take a while as I have literally no idea how to do that now...